README.md

```diff
@@ -218,10 +218,12 @@ The following inductive datasets are built in to PyKEEN.
 | TuckER                         | [`pykeen.models.TuckER`](https://pykeen.readthedocs.io/en/latest/api/pykeen.models.TuckER.html)                                       | [`pykeen.nn.TuckerInteraction`](https://pykeen.readthedocs.io/en/latest/api/pykeen.nn.TuckerInteraction.html)                              | [Balažević *et al.*, 2019](https://arxiv.org/abs/1901.09590)                                                            |
 | Unstructured Model             | [`pykeen.models.UM`](https://pykeen.readthedocs.io/en/latest/api/pykeen.models.UM.html)                                               | [`pykeen.nn.UMInteraction`](https://pykeen.readthedocs.io/en/latest/api/pykeen.nn.UMInteraction.html)                                      | [Bordes *et al.*, 2014](https://link.springer.com/content/pdf/10.1007%2Fs10994-013-5363-6.pdf)                          |
 
-### Losses (14)
+### Losses (16)
 
 | Name                                 | Reference                                                                                                                       | Description                                                                                           |
 |--------------------------------------|---------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------|
+| adversarial                          | [`pykeen.losses.AdversarialLoss`](https://pykeen.readthedocs.io/en/latest/api/pykeen.losses.AdversarialLoss.html)               | A loss with adversarial weighting of negative samples.                                                |
```

Comment:
this shouldn't appear in the README

Reply:
b93c0c3af79c4719c5da9e878cc42abb74f7aafc...38b63325b464037a1cd2dbb9235d6fb329467c8a